src/Microsoft.ML.TensorFlow/TensorflowCatalog.cs

```diff
@@ -11,6 +11,26 @@ namespace Microsoft.ML
     /// <include file='doc.xml' path='doc/members/member[@name="TensorflowTransformer"]/*' />
     public static class TensorflowCatalog
     {
+        /// <summary>
+        /// Scores a dataset using a pre-traiend TensorFlow model located in <paramref name="modelLocation"/>.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="modelLocation">Location of the TensorFlow model.</param>
+        /// <param name="inputColumnName"> The name of the model inputs.</param>
+        /// <param name="outputColumnName">The name of the requested model outputs.</param>
+        /// <example>
```

Comment:
Zeeshan A is trying to rename this file, so be careful
[https://github.com/dotnet/machinelearning/pull/2429](https://github.com/dotnet/machinelearning/pull/2429)